src/widgets/table.rs

```diff
@@ -194,6 +194,21 @@ impl HighlightSpacing {
     }
 }
 
+#[derive(Debug, Display, EnumString, PartialEq, Eq, Clone, Default, Hash)]
+pub enum HighlightArea {
+    /// No highlight
+    None,
+    /// Highlight cell only
+    #[default]
+    Cell,
+    /// Highlight entire row
+    Row,
+    /// Highlight entire column
+    Col,
+    /// Highlight entire row and column
+    RowAndCol,
```

Comment:
Is this likely to ever have any other values? What do you think about calling this `Both` instead of `RowAndCol`?

This is not a blocker - I'm actually unsure which name works better.

Reply:
I thought about this for a while, and decided that `RowAndCol` was more explicit, and thus better IMO. I came to this conclusion when I thought about what "HighlightArea::Both" would look like if seen out of context, vs "HighlightArea::RowAndCol".. 

I also thought about more values, and while adding a #[non_exhaustive] should shield us from having to break code in the future, I also feel that we could decide now if we want more options.

Other than `None`, `Cell`, `Row`, `Col`, and `RowAndCol`, I thought about:

- Row without Cell  
- Col without Cell
- RowAndCol without Cell

But none of them seem that relevant to me, and would make the implementation considerably more convoluted, and so I dropped them. If you think these or any other values should be added, I can definitely work on it!

Reply:
Sounds like you chose right already with RowAndCol. I'm going to resolve this one for now.